### PR TITLE
Fixes and updates in docs and HTML files

### PR DESCRIPTION
**Pull request by cwonrails**
There are still issues with the various Bootstrap links throughout `README.md`, but several others were addressed. The HTML fixes are pretty important though IMO. Thanks for this very useful repo!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ we have found that:
   + Lots of **Duplication** _where people re-create styles over-and-over_
   + "**_Zombie_ Code**" _unused styles that nobody will risk removing_
   + **Bloated** apps/sites that take _much_ longer
-  to develop, load and maintain than they _neeD_ to.
```
Comment (nelsonic) [inline README.md:77]: thanks for fixing this typo. I saw it too while re-reading today ... 👍

File: README.md [hunk 2]
```diff
@@ -325,7 +325,7 @@ Deleting unused css is hard and time consuming.
 And more often than not - it’s not work people
 are excited to do. So what happens?
 People keep writing more and more css -
-[Adam Morse](http://mrmrs.io/writing/2016/03/24/scalable-css/).
+[Adam Morse](http://mrmrs.github.io/writing/2016/03/24/scalable-css/).
```
Comment (nelsonic) [inline README.md:328]: 👍

File: README.md [hunk 3]
```diff
@@ -472,7 +472,7 @@ You can get up and running by just
 adding a link tag in your html
 
 ```html
-<link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css">
+<link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.1/css/tachyons.min.css">
```
Comment (nelsonic) [inline README.md:475]: 👍

File: examples/buttons.html [hunk 4]
```diff
@@ -1,8 +1,11 @@
+<!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```
Comment (nelsonic) [inline examples/buttons.html:3]: Firefox dev complains when there's no content type. 
But otherwise no issue with the removal.

**Review by nelsonic — APPROVED** [review]
@cwonrails thanks _very_ much for making time to make these updates. 🎉

**Review by iteles — COMMENTED** [review]
Thanks for this @cwonrails, all good spots, glad you find this repo useful! :tada:

If you haven't already seen it, https://github.com/dwyl/tachyons-bootstrap may also be of some use to you 👍 :blush:

_Welcome to dwyl!_

**Comment by cwonrails** [comment]
Thanks for the hearty welcome! Will do what I can to contribute as I’m able!

**Pull request merged**